layers\vulkan\generated/vk_validation_error_messages.h has wrong encoding

There are a large number of unpresentable character encoding errors in layers\vulkan\generated/vk_validation_error_messages.h which cause the build to fail

```
2>layers\vulkan\generated/vk_validation_error_messages.h(327,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(1373,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(2441,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(2724,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(5566,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(6225,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(6352,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(7534,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(8174,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(10043,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(10274,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(10355,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(10651,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(10730,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(11038,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(11716,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(11796,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(12392,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(12470,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(12768,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(13065,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(13439,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(14495,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(16268,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
2>layers\vulkan\generated/vk_validation_error_messages.h(17961,1): warning C4819: The file contains a character that cannot be represented in the current code page (932). Save the file in Unicode format to prevent data loss (compiling source file layers\error_message\logging.cpp)
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -137,6 +137,7 @@ elseif(MSVC)
 
     # Enforce stricter ISO C++
     add_compile_options(/permissive-)
+    add_compile_options(/utf-8)
 
     # PDBs aren't generated on Release builds by default.
     add_compile_options("$<$<CONFIG:Release>:/Zi>")

